README.md

```diff
@@ -1,25 +1,24 @@
-# Phyloreferencing Curation Workflow
-The Phyloreferencing curation workflow serves three main purposes:
+# Clade Ontology
 
-1. It provides a set of exemplar curated phyloreferences in JSON and OWL.
-2. It provides a test space for trying different approaches to generating phyloreferences from JSON to OWL, although these will be moved into their own repositories if they prove to be successes.
-3. It provides a test suite of phyloreferences along with expected resolved nodes, allowing reasoning to be continually tested as ontologies and software tools are updated.
+The Clade Ontology is an ontology of exemplar phyloreferences curated from peer-reviewed publications. Phyloreferences in this ontology include their verbatim clade definition and the phylogeny upon which they were initially defined. The ontology therefore acts as both a catalogue of computable clade definitions as well as a test suite of phyloreferences that can be tested to determine if each phyloreference resolves as expected. This ontology is expressed in the [Web Ontology Language (OWL)](https://en.wikipedia.org/wiki/Web_Ontology_Language). The executable software code in this repository is available for reuse under the terms of the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0).
 
```

Comment:
My feeling is we should standardize on a "default" software license, because I don't see much point in having different licenses for different pieces of software we develop, given that we're not defaulting on the GPL, which we may otherwise have reason to relax from depending on the specific tool or code.

So are you suggesting the Apache 2.0 license as our default, and do you have reasons to choose it over, say MIT, or BSD2/3, as the most commonly used "permissive" licenses.

Reply:
Actually, it looks we've standardized to the MIT license across all Phyloref software products as we decided to in phyloref/phylo2owl#1 -- *except* for the Clade Ontology, which ended up with the Apache License before we'd closed that issue. I've now relicensed it with the MIT license in 5e1ec27.